packages/manager/modules/pci/src/projects/project/data-processing/home/data-processing-home.html

```diff
@@ -69,7 +69,7 @@
                         <oui-tile-actions>
                             <button
```

Comment:
```suggestion
                            <oui-button
```

Reply:
Linked to https://github.com/ovh/manager/pull/8470#discussion_r1044496268